mara_db/shell.py

```diff
@@ -5,10 +5,11 @@
 """
 
 import shlex
-import sys
 from functools import singledispatch
 
+import sys
 from mara_db import dbs, config
+from mara_db.bigquery import bigquery_credentials
```

Comment:
It would be nice if this could be `import mara_db.bigquery as bq` and then the usage as `bg.bigquery_credentials(...)` -> in our setting we have to import the credentials via secrets and so this is not a file. Calling it via the module allows patching.

Reply:
... but if I read the below right, you anyway need it in the defaul location for `bq load` to work?

Reply:
Yes. I tried hard to get it working without a file, and failed.

Reply:
We have now suspicious manually copied credentials files flying around, not nice